docs/dml-of-carbondata.md

```diff
@@ -352,18 +352,20 @@ CarbonData DML statements are documented here,which includes:
   ```
   LOAD DATA [LOCAL] INPATH 'folder_path' 
   INTO TABLE [db_name.]table_name PARTITION (partition_spec) 
-  OPTIONS(property_name=property_value, ...)    
-  INSERT INTO INTO TABLE [db_name.]table_name PARTITION (partition_spec) <SELECT STATEMENT>
+  OPTIONS(property_name=property_value, ...);
```

Comment:
The sqls all of this file don't have ';',  why you only add some of sqls
I also think it's unnecessary to add ';',   only beeline needs  and CarbonSession doesn't
Only blank is OK

Reply:
done